### PR TITLE
[MB-8856] refactor prime api demo script

### PR DESCRIPTION
**Pull request by pearl-truss**
## Description

This PR refactors the `prime-api-demo` script into the `pricing-acceptance` script that works in the same way. The big difference is that this script creates a new shipment so instead of starting with customer onboarding, you can start with an existing move that has been approved (you'll need either the move id or move locator code)

## Reviewer Notes

I discovered that the `create-upload` endpoint is broken while working on this. The CLI works without uploading a document when creating a payment request. However, I will create a ticket to update the script once the create-upload endpoint is fixed

## Setup

```sh
pricing-acceptance RDY4PY local
```
You will need to create a payload for the service item that you want to create and price and will be prompted by the CLI to input the path to this payload json file

When prompted you will need to sign in to the office view as a TIO to approve the payment request

Before the script exits, you should see an EDI generated for the service item you created

## Acceptance
Run this CLI against staging (first find an approved move in staging and pass the move code or ID to the cli)

```
pricing-acceptance RDY4PY api.stg.move.mil
```

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-8856) for this change
* [documentation](https://github.com/transcom/mymove/wiki/Acceptance-Testing-Payment-Requests#pricing-acceptance-tool) for this change

### CODE REVIEW

File: scripts/pricing-acceptance [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+}
+EOM
+
+bin/prime-api-client "${primeapiopts[@]}" update-mto-shipment --filename ./tmp/update_mto_shipment.json > tmp/update_response_mto_shipment.json
+
+# --------------------
+# CREATE SERVICE ITEM
+shipmentID=$(jq '.id' tmp/update_response_mto_shipment.json)
+
+printf "Use the shitpment and moveTaskOrder IDs found below to create a payload for a new service item:\n\n"
```
Comment (reggieriser) [inline scripts/pricing-acceptance:185]: Nit: typo in `shitpment`.

File: scripts/pricing-acceptance [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+# script to help with prime demo script
+#
+
+# =====================================
+#     USAGE AND DEFAULTS
+# =====================================
+set -eu -o pipefail
+proofs=(./tmp/uploads/proof_of_service.pdf)
+function usage() {
+  echo "Usage: prime-api-demo <movecode> [ <hostname> [ <proofs> ... ] ]"
```
Comment (reggieriser) [inline scripts/pricing-acceptance:13]: Usage still has a `prime-api-demo` reference here.

**Review by reggieriser — APPROVED** [review]
Found a couple of very minor things, but otherwise this LGTM.  I was able to follow the instructions and create a service item and see EDI in the end -- much quicker than before! 🚀 

Note that there are still a few `prime-api-demo` references ([first](https://github.com/transcom/mymove/wiki/Manually-run-Prime-API-for-Slice-demo), [second](https://github.com/transcom/mymove/wiki/Testing-payment-requests-for-domestic-SIT-service-items)) in the Wiki that we should probably clean up since that script is being deleted here.

File: scripts/pricing-acceptance (hunk 1 above)
Comment (pearl-truss) [reply scripts/pricing-acceptance]: ha thanks for catching this!

**Comment by pearl-truss** [comment]
> Found a couple of very minor things, but otherwise this LGTM. I was able to follow the instructions and create a service item and see EDI in the end -- much quicker than before! 🚀
> 
> Note that there are still a few `prime-api-demo` references ([first](https://github.com/transcom/mymove/wiki/Manually-run-Prime-API-for-Slice-demo), [second](https://github.com/transcom/mymove/wiki/Testing-payment-requests-for-domestic-SIT-service-items)) in the Wiki that we should probably clean up since that script is being deleted here.

Thanks, I made the code and documentation changes

**Pull request merged**